lib/dealership.rb

```diff
@@ -0,0 +1,34 @@
+class Dealership
+  attr_reader :dealership,
+              :address,
+              :inventory
+
+  def initialize(dealership, address)
+    @dealership   = dealership
+    @address      = address
+    @inventory    = []
+  end
+
+  def inventory_count
+    @inventory.count
+  end
+
+  def add_car(car)
+    @inventory.push(car)
+  end
+
+  def cars_by_make(make)
+    @inventory.select do |invent|
+      invent.cars.make == make
+    end
+  end
```

Comment:
This is close. When you call `.select` you are iterating through each object within the @inventory array. You have access to that object via the variable defined between `| |` so for you that is `invent` and `invent` is the Car object. You can then call make on the car:

```suggestion
  def cars_by_make(make)
    @inventory.select do |invent|
      invent.make == make
    end
  end
```